image/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM fedora
```

Comment:
I can see how this can be your preference. :D But I think `busybox` should be fine for now?

Reply:
You clearly have excellent diplomatic skills :) yes, busybox is totes fine.